constraint/pkg/client/drivers/k8scel/schema/schema.go

```diff
@@ -22,6 +22,8 @@ const (
 	ReservedPrefix = "gatekeeper_internal_"
 	// ParamsName is the VAP variable constraint parameters will be bound to.
 	ParamsName = "params"
+	// ObjectName is the VAP variable that describes either an object or (on DELETE requests) oldObject
+	ObjectName = "anyObject"
```

Comment:
With this change, do we plan to update all cel expressions for gatekeeper-libraries policies to use this before merging those PRs? if so, then we need to cut a patch release for GK and update GK-lib CI and docs to ensure the latest framework/GK is used.

Reply:
yea, we'll need to merge this, vendor it in GK, cut a GK release (3.16.1), and update library